composer.json

```diff
@@ -11,6 +11,8 @@
     ],
     "require": {
         "php": ">=7.1.0",
+        "ext-json": "*",
```

Comment:
Now that `legacy.json` is stored in an array, we can actually remove `ext-json`.

Reply:
I noticed some other usages in the library so I'm gonna leave it in.